features/manage-position/YourPosition.tsx

```diff
@@ -121,37 +121,40 @@ const YourPosition = () => {
           {`${_collateral} ${_collSymbol}`}
         </Status>
         <Status>
-          <Label>Tokens outstanding: </Label>
+          <Label>Tokens created: </Label>
           {`${_tokens} ${_tokenSymbol}`}
         </Status>
         <Status>
           <Label>
-            Estimated identifier price (
+            Identifier price(
             <Link href={pricefeedUrl} target="_blank" rel="noopener noreferrer">
-              Coinbase Pro
+              Coinbase
             </Link>
             ):{" "}
           </Label>
-          {_latestPrice}
+          {Number(_latestPrice).toFixed(4)}
         </Status>
         <Status>
-          <Label>(CR) Collateralization ratio:</Label>
-          {` ${pricedCR} (${_collSymbol} / ${_tokenSymbol})`}
+          <Label>Collateralization ratio(CR):</Label>
```

Comment:
There should be a space between "ratio" and "(CR)"

Reply:
I removed this intentionally to save space for mobile screen sizes but can add it back.

Reply:
I'd like to add it back they shouldn't be in the same word (without spaces between)